src/Aspire.ProjectTemplates/templates/aspire-starter-net8/Aspire-StarterApplication.1.ApiService/Properties/launchSettings.json

```diff
@@ -4,8 +4,7 @@
     "http": {
       "commandName": "Project",
       "dotnetRunMessages": true,
-      "launchBrowser": true,
-      "launchUrl": "weatherforecast",
+      "launchBrowser": false,
```

Comment:
Is this what we have now in our .NET 8 templates?

Reply:
This is what the ASP.NET Core 9 templates have now, yes. I figured it's best to align rather than do something different.